src/display/annotation_layer.js

```diff
@@ -239,6 +239,40 @@ class AnnotationElement {
     return container;
   }
 
+  /**
+   * Create areas for the quadPoints.
+   *
+   * @private
+   * @memberof AnnotationElement
```

Comment:
`_createContainer` returns the container, with `@returns {HTMLSectionElement}` added here, but this function set the member variable. For consistency it would be better to also let this method return a list of quadrilaterals, or `null` if not applicable.